pysignalscope/scope.py

```diff
@@ -853,7 +921,801 @@ def plot_channels(*channel: List['Scope'], timebase: str = 's', figure_size: Opt
+        Called by mathplotlib-Event and assigned to an update of text.
+
+        :param text: Updated text
+        :type text: string
+        """
+        # Check shift direction
+        if HandleScope.shift_dir == 1:
+            HandleScope.shiftstep_y = float(text)
+            if HandleScope.shiftstep_y > HandleScope.max_shiftstep_y:
+                # Shift step x too high
+                logging.info(f"{class_modulename} :Shift step y {HandleScope.shiftstep_y} too high and set to {HandleScope.max_shiftstep_y}")
+                # Correct the value
+                HandleScope.shiftstep_y = HandleScope.max_shiftstep_y
+                HandleScope.shift_text_box.set_val(str(HandleScope.shiftstep_y))
+                # Log flow control
+                logging.info(f"{class_modulename} :Shift step y too small.{HandleScope.shiftstep_y}")
+            elif HandleScope.shiftstep_y < HandleScope.min_shiftstep_y:
+                # Shift step x too small
+                logging.info(f"{class_modulename} :Shift step y {HandleScope.shiftstep_y} too small and set to {HandleScope.min_shiftstep_y}")
+                # Correct the value
+                HandleScope.shiftstep_y = HandleScope.min_shiftstep_y
+                HandleScope.shift_text_box.set_val(str(HandleScope.shiftstep_y))
+
+            # Log flow control
+            logging.debug(f"{class_modulename} :Shift step y = {HandleScope.shiftstep_y}")
+
+        else:
+            HandleScope.shiftstep_x = float(text)
+            if HandleScope.shiftstep_x > HandleScope.max_shiftstep_x:
+                # Shift step x too high
+                logging.info(f"{class_modulename} :Shift step x {HandleScope.shiftstep_y} too high and set to {HandleScope.max_shiftstep_x}")
+                # Correct the value
+                HandleScope.shiftstep_x = HandleScope.max_shiftstep_x
+                HandleScope.shift_text_box.set_val(str(HandleScope.shiftstep_x))
+            elif HandleScope.shiftstep_x < HandleScope.min_shiftstep_x:
+                # Shift step x too small
+                logging.info(f"{class_modulename} :Shift step x {HandleScope.shiftstep_x} too small and set to {HandleScope.min_shiftstep_x}")
+                # Correct the value
+                HandleScope.shiftstep_x = HandleScope.min_shiftstep_x
+                HandleScope.shift_text_box.set_val(str(HandleScope.shiftstep_x))
+
+        # Log flow control
+        logging.debug(f"{class_modulename}: Shift step x = {HandleScope.shiftstep_x},Shift step y = {HandleScope.shiftstep_y}")
+
+        # Reset the values
+        HandleScope.last_val = 0
+        HandleScope.shift_slider.set_val(0)
+        # Update the plot
+        HandleScope.shiftfig.canvas.draw_idle()
+
+        # Log flow control
+        logging.debug(f"{class_modulename} :Text is overtaken {HandleScope.shift_dir} (0=x, 1=y).")
+
+    # Callback-Funktion for shift the channel by movement of the slider
+    @staticmethod
+    def shiftchannel(val: float):
```

Comment:
inconsistent method naming, other methods are using underscores -> shift_channel

Reply:
Yes this is indeed the case.
Issues are corrected.